api-specs/Gateway-EE/latest/kong-ee.yaml

```diff
@@ -14762,6 +15132,9 @@ tags:
   - description: |
       Admin routes
     name: admins
+  - description: |
+      Group routes
+    name: groups
   - description: |
       Debug Routes
     name: debug
```

Comment:
```suggestion
  - description: |
      Admin routes
    name: Admins
  - description: |
      Group routes
    name: Groups
  - description: |
      Debug Routes
    name: Debug
```

Shall we capitalize tag names to keep consistent with other tags?

<img width="654" alt="Screenshot 2024-06-03 at 16 19 27" src="https://github.com/Kong/docs.konghq.com/assets/5277268/015112e1-8c69-49dd-81a1-9576ecbd7d5c">